nuxeo-platform-video-core/src/main/java/org/nuxeo/ecm/platform/video/tools/operations/ExtractClosedCaptionsFromVideo.java

```diff
@@ -49,7 +45,7 @@
 
     public static final String ID = "Video.ExtractClosedCaptions";
 
-    @Param(name = "outFormat", required = false, values = { "txt" })
+    @Param(name = "outFormat", required = false, values = { "ttxt", "srt", "txt" })
```

Comment:
The goal was to expose the output formats that were supported by the operation. Is there an alternative to do this?